base/database/testing.go

```diff
@@ -312,3 +314,56 @@ func UpdateSystemAdvisoriesWhenPatched(t *testing.T, systemID, accountID int, ad
 		Update("when_patched", whenPatched).Error
 	assert.Nil(t, err)
 }
+
+func CreateBaseline(t *testing.T) int {
+	type BaselineConfig struct {
+		ToTime string `json:"to_time"`
+	}
+
+	baselineConfig, err := json.Marshal(BaselineConfig{ToTime: "2021-01-01 12:00:00-04"})
+	assert.Nil(t, err)
+
+	temporaryBaseline := &models.Baseline{
+		RhAccountID: 1, Name: "temporary_baseline", Config: baselineConfig,
+	}
+	err = Db.Create(temporaryBaseline).Error
+
+	assert.Nil(t, err)
+	checkBaselineStatus(t, temporaryBaseline.ID, 1)
+
+	return temporaryBaseline.ID
+}
+
+func DeleteBaseline(t *testing.T, baselineID int, inventoryIDs []string) {
```

Comment:
You don't need to specify inventoryIDs here. Just take all inventory items having baseline_id set to given baseline id